extensions/vertx-http/runtime/src/test/java/io/quarkus/vertx/http/runtime/DefaultAccessLogReceiverTest.java

```diff
@@ -0,0 +1,453 @@
+package io.quarkus.vertx.http.runtime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.concurrent.Executor;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import io.quarkus.vertx.http.runtime.filters.accesslog.DefaultAccessLogReceiver;
+import io.quarkus.vertx.http.runtime.filters.accesslog.LogFileHeaderGenerator;
+
+class DefaultAccessLogReceiverTest {
+
+    @TempDir
+    Path tempDir;
+    private Executor testExecutor;
+    private DefaultAccessLogReceiver receiver;
+
+    @BeforeEach
+    void setUp() {
+        testExecutor = command -> command.run();
+    }
+
+    @AfterEach
+    void tearDown() throws IOException {
+        if (receiver != null) {
+            receiver.close();
+        }
+    }
+
+    @Test
+    void testBasicLogging() throws Exception {
+        receiver = new DefaultAccessLogReceiver(testExecutor, tempDir.toFile(), "test-basic");
+        receiver.logMessage("Test message");
+
+        // Wait briefly for async processing
+
+        Path logFile = tempDir.resolve("test-basic.log");
+        assertTrue(Files.exists(logFile));
+        String content = Files.readString(logFile, StandardCharsets.UTF_8);
+        assertTrue(content.contains("Test message"));
+    }
+
+    @Test
+    void testMultipleMessages() throws Exception {
+        receiver = new DefaultAccessLogReceiver(testExecutor, tempDir.toFile(), "test-multi");
+
+        for (int i = 0; i < 10; i++) {
+            receiver.logMessage("Message " + i);
+        }
+
+        Path logFile = tempDir.resolve("test-multi.log");
+        String content = Files.readString(logFile, StandardCharsets.UTF_8);
+        for (int i = 0; i < 10; i++) {
+            assertTrue(content.contains("Message " + i));
+        }
+    }
+
+    @Test
+    void testLogRotation() throws Exception {
+        receiver = new DefaultAccessLogReceiver(testExecutor, tempDir.toFile(), "test-rotate");
+
+        // Create initial log
+        receiver.logMessage("First message");
+
+        // Force rotation
+        receiver.rotate();
+
+        // Write after rotation
+        receiver.logMessage("Second message");
+
+        // Verify both files exist
+        assertTrue(Files.exists(tempDir.resolve("test-rotate.log")));
+        assertTrue(Files.list(tempDir)
+                .anyMatch(p -> p.getFileName().toString().matches("test-rotate\\d{4}-\\d{2}-\\d{2}\\.log")));
```

Comment:
This indicates to me that the issue is not fixed.

Reply:
will check again. thanks.